etc/schema/mysql-upgrades/2.11.0.sql

```diff
@@ -21,7 +21,13 @@ CREATE TABLE icingaweb_schema (
   timestamp int unsigned NOT NULL,
 
   PRIMARY KEY (id)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin ROW_FORMAT=DYNAMIC;
+
+ALTER TABLE `icingaweb_group` ROW_FORMAT=DYNAMIC;
```

Comment:
This has to be before the convert to `utf8mb4`.

Reply:
Fixed